### PR TITLE
Add Prometheus metrics

### PR DESCRIPTION
**Pull request by AminArria**
## Summary of changes

- Add dependencies to have prometheus metrics and expose them in http://localhost:9568/metrics
- Add some basic in-game metrics to track tick duration, game count, and player count

## How to test it?

Setup Grafana dashboards
1. Spin up Prometheus and Grafana docker containers (`docker compose up -d`)
2. Turn on server (`make start` or whatever you use)
3. Go to http://localhost:9100 and log in, default user and password is `admin`
4. In sidebar go to `Connections > Data sources`
  4.1.  For `Prometheus server URL` set it to `http://prometheus:9090`
  4.2  Click on `Save & test`
5. In Sidebar go to `Dashboards`
  5.1. Click on `New` to show dropdown and select `Import`
  5.2 In the `Import via dashboard JSON model` section copy & paste the contents of `grafana_arena_dashboard.json` (file is in this PR) 
  5.3 Click on `Load`
  5.4 In the `prometheus` section pick your recently created data source
  5.5 Click `Import`

At this point you should be seeing some empty dashboards, now play a game (or more) and you should start to see data come into the dashboards

## Checklist
- [x] Tested the changes locally.
- [ ] Reviewed the changes on GitHub, line by line.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.

### CODE REVIEW

**Review by tvillegas98 — APPROVED** [review]
Great addition! let's create an issue to address and improve the chart for the game tick duration

**Comment by AminArria** [comment]
> Great addition! let's create an issue to address and improve the chart for the game tick duration

https://github.com/lambdaclass/mirra_backend/issues/829#issue-2445040232

File: docker-compose.yml
[the remark this answers is not in the document]
Comment (tvillegas98) [reply docker-compose.yml:1]: But lets create an issue to address in devenv

**Review by agustinesco — APPROVED** [review]
LGTM!

**Pull request merged**